apps/tlon-mobile/src/types.ts

```diff
@@ -41,12 +41,34 @@ export type RootStackParamList = {
     post: db.Post;
     uri?: string;
   };
+  GroupSettings: {
+    group: db.Group;
+  };
+};
+
+export type GroupSettingsStackParamList = {
+  GroupMeta: {
+    group: db.Group;
```

Comment:
Somewhat random, but I've found it occasionally annoying to always need a full model when routing. If we're usually going to be pulling the group off `useGroupContext` anyhow, wonder if we could get away just passing the `groupId`

Fine as is if you'd prefer, just a stray thought

Reply:
Yeah, agreed. I'll change it.